src/components/FilterPanel/index.js

```diff
@@ -116,10 +116,13 @@ export default function FilterPanelParent({
                                 className={classes.mobileButton}
                                 onClick={() => setFilters(inProgressFilters)}
                                 disabled={
-                                    inProgressFilters.filterByZipCode.zipCode &&
-                                    !isZipValid(
+                                    !!(
```

Comment:
@harcod did you mean to do this? idk how this is related to the change at hand

Reply:
(Unrelated but...) Yes.  It was required to pass a jest test.  It wouldn't let me commit!!

Reply:
This change forced it to be a 'boolean' that jest was complaining about.